### ISSUE TITLE
Markdown support for Toothpaste

### ISSUE THREAD

**Issue by agisilaos** [imcatnoone/toothpaste#64]
It seems like that markdown support for Toothpaste is missing atm. It would be really cool if the theme can support the `.md` syntax. 

Here's the current situation of toothpaste when it handles `.md` files

![screen shot 2017-10-24 at 14 48 28](https://user-images.githubusercontent.com/10944108/31941501-c09d7934-b8ca-11e7-8dca-8fd6f2d9348a.png)

The `One-Dark` theme in Atom support this format and it looks pretty rad

![screen shot 2017-10-24 at 14 44 13](https://user-images.githubusercontent.com/10944108/31941255-e228b718-b8c9-11e7-93d9-49e6feab7b70.png)

**Comment by joshgoebel** [comment]
Yes, please.  If Toothpaste still actively maintained?  If a PR was created for this, would it be merged?
